src/docs/projects/staticanalysis-bot.rst

```diff
@@ -5,29 +5,29 @@ Project: staticanalysis/bot
 
 :contact: `Bastien Abadie`_, (backup `Release Management`_)
 
-Static analysis bot is a Taskcluster task, triggered by *Pulse Listener* on every new Mozreview or Phabricator patch.
-The task applies several code analyzers on each patch:
+Static analysis bot is a set of Taskcluster tasks, triggered by *Pulse Listener* on every new Phabricator patch, using a `HarborMaster build plan`_.
```

Comment:
It sounds strange to say `Static Analysis bot is a set of Taskcluster tasks`. Maybe `Static Analysis bot is composed of a set of Taskcluster tasks`